config/routes.rb

```diff
@@ -0,0 +1,26 @@
+Rails.application.routes.draw do
+  root "pages#home"
+
+  get '/foods', to: 'foods#index'
+  get '/recipes', to: 'recipes#index'
+  get '/shopping_lists', to: 'shopping_lists#index'
+
+
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
+
+  #get '/shopping_lists', to: 'shopping_lists#index'
+
+  resources :users, only: [:index, :show] do
+    resources :foods, only: [:index, :new, :create, :destroy]
+  end
+  resources :recipes do
+    resources :recipe_foods
+  end
+  
+  resources :public_recipes
```

Comment:
[REQUIRED] Kindly restrict your public_recipes endpoint to only index since you have only index in your PublicRecipes controller